webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -86,8 +89,8 @@ const TrialChart: React.FC<Props> = ({
       },
       series: [
         { label: 'Batch' },
-        ...metrics.map((metricName, index) => ({
-          label: metricName.name,
+        ...metrics.map((metric, index) => ({
+          label: `${metric.name} (${metric.type})`,
```

Comment:
suggestion: if we can use any elements here it might be good to try and keep it consistent with how we display the type in other places  
![2021-07-07_15-09-49](https://user-images.githubusercontent.com/12127420/124835001-7985ba80-df35-11eb-93fb-401e5bd81a7d.png)

if it's text only maybe `[V] metricname` ?

Reply:
@apizzini thoughts on this? I agree that this would keep it consistent and make it easier for the user to connect the dots when they see the prefix. Visual div prefix would be preferred, but a text `[V]` or `[T]` is acceptable as well.

Reply:
Done :)